src/coordinator/commands/time_register_ops.rs

```diff
@@ -134,6 +134,22 @@ impl SetTimeRegister {
         self.set_register(self.action.register()? + 1, &self.values[2..4])
             .await?;
 
+        // FIXME: If we only update one of the two registers, we should probably
```

Comment:
This change publishes a `lxp/XX/ac_charge/1 {"start":"00:00","end":"23:59"}` message for us when the command is run. Without it, Home Assistant assumes the attempt was a failure and resets the text field back to the old value after ?some time? ?a restart of lxp-bridge?

The changes to the underlying register are already published, but of course, we don't use them in `state_topic`.